docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -737,6 +737,49 @@ The output of the query has the following columns:
     - ``boolean``
     - Whether or not the operation appended data
 
+Functions
+---------------
+
+The connector exposes following functions:
+
+table_changes
+^^^^^^^^^^^^^
+
+Allows reading Change Data Feed (CDF) entries. This feature allows Delta tables to track row-level changes between versions of a Delta table.
+When enabled on a Delta table, it records "change events" for all the data written into the table.
+
+.. code-block:: sql
+
+    SELECT * FROM TABLE(system.table_changes(schema_name => 'test_schema', table_name => 'tableName', since_version => 0))
+
+``since_version`` - type ``BIGINT``, exclusive, changes will be read from ``since_version + 1``
+
+If ``since_version`` is not provided  entire history will be read.
+Example:
+
+.. code-block:: sql
+
+    CREATE TABLE pages (page_url VARCHAR, domain VARCHAR, views INTEGER) WITH (change_data_feed_enabled = true);
+    INSERT INTO pages VALUES('url1', 'domain1', 1), ('url2', 'domain2', 2), ('url3', 'domain1', 3);
+    INSERT INTO pages VALUES('url4', 'domain1', 400), ('url5', 'domain2', 500), ('url6', 'domain3', 2);
+    UPDATE pages SET domain = 'domain4' WHERE views = 2;
+    SELECT * FROM TABLE(system.table_changes('test_schema', 'pages');
```

Comment:
should we offer the users here also `ORDER BY` in the query so that the changes can be read in order?

Reply:
you mean in the doc? sure, it is supported I will add it to one test